baseapp/pom.xml

```diff
@@ -15,8 +15,8 @@
     <packaging>jar</packaging>
 
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.9</maven.compiler.source>
```

Comment:
Why are we changing to Java 1.9  .. ultimately will change to 17 but is this change necessary?

Reply:
this can stay 1.8, but we will need 1.9 for gateway-ha because last time it failed when I built it with 1.8.